src/api/fileApi.ts

```diff
@@ -87,11 +87,11 @@ export const listParts = async (uploadId: string, key: string) => {
   return listPartsResponse.data;
 };
 
-export const prepareUploadPart = async (uploadId: string, key: string, body: Blob, number: number) => {
+export const signPart = async (uploadId: string, key: string, number: number) => {
   const payload = {
     uploadId,
     key,
-    body: JSON.stringify(body),
+    body: JSON.stringify('ignored'),
     number,
   };
 
```

Comment:
eller hæ... dette kallet går jo fortsatt mot `/prepare` ser jeg 🤯 
måtte du ikke lage nytt endepunkt for `signParts` eller noe?

Reply:
Gjerne kjør en runde lokalt og se om det fungerer til deg også.

Reply:
hmm, lurer på om jeg har blinksa hardt på dette :D 

vi har egne endepunkter i for alle metodene nedover her: https://uppy.io/docs/aws-s3-multipart#createmultipartuploadfile
og der kom det noen endringer i v2, men det ser kanskje ut som man kan gjenbruke det gamle endepunktet for `prepareUpload` for den nye `signPart`... Men da tipper jeg det er litt avvik mellom nøyaktig hva endepunktene skal gjøres vs hva de gjør i dag 🤔

Reply:
men ok, da har jeg åpenbart misfortått litt her ja og vi kan bare merge. Men vi burde gå gjennom litt å sjekke at alt er i sync her da, rename paths og sjekke at det det sendes riktig data (begge veier) og at vi ikke baseres oss på et luftslott her

Reply:
Tror det er uppy dokumentasjonen som er crap. Da jeg skulle implementere backenden var det eneste jeg hadde å gå på var et nodeJs eksempel og stackoverflow.

Reply:
Tror preparePart og signedPart i uppy ble endret fordi de syntes det var merkelig med POST fetchSignedUrl